plugin.xml

```diff
@@ -87,6 +87,9 @@
 
         <header-file src="src/ios/UNNotificationRequest+APPLocalNotification.h" />
         <source-file src="src/ios/UNNotificationRequest+APPLocalNotification.m" />
+
+        <resource-file src="src/sounds/komed_notification_default.wav" target="komed_notification_default.wav"/>
```

Comment:
so you decided to add these resource files after all in this plugin and not the komed-plugin-push?

Reply:
yes. I think it's better to have the reference to file in the same repo as the file